Applications/DataExplorer/VtkVis/VtkRaster.cpp

```diff
@@ -309,19 +309,18 @@ std::string VtkRaster::findWorldFile(std::string const& filename)
 {
     std::string const no_ext = BaseLib::dropFileExtension(filename);
 
-    std::vector<std::string> const supported_extensions =
-    { ".pgw", ".pngw", ".pgwx",
-      ".jgw", ".jpgw", ".jgwx",
-      ".tfw", ".tifw", ".tfwx",
-      ".bpw", ".bmpw", ".bpwx",
-      ".wld" };
-
-    for (auto& ext : supported_extensions)
+    std::array<std::string, 13> const supported_extensions = {
```

Comment:
style: It is no longer necessary to give the template arguments. Though the type changes to `std::array<char*, 13>`, which is good, but maybe expected somewhere else to be a string container.
```suggestion
    constexpr std::array supported_extensions = {
```